src/main/treeDataProviders/dependenciesTree/dependenciesRoot/npmTree.ts

```diff
@@ -23,17 +23,25 @@ export class NpmTreeNode extends RootNode {
             projectDetails.loadProjectDetails(NpmCmd.runNpmLs(this.workspaceFolder));
         } catch (error) {
             this._logManager.logError(<any>error, false);
+            projectDetails.loadProjectDetails(JSON.parse((<any>error).stdout.toString()));
```

Comment:
Looks like `loadProjectDetailsFromFile` overrides the package name and version assignment you did in loadProjectDetails.

Please consider the following:
```typescript
        let npmLsResult: string;
        try {
            npmLsResult = NpmCmd.runNpmLs(this.workspaceFolder);
        } catch (error) {
            this._logManager.logError(<any>error, false);
            npmLsResult = JSON.parse((<any>error).stdout.toString());
            npmLsFailed = true;
        }
        projectDetails.loadProjectDetails(npmLsResult);
```